src/main/java/seedu/duke/tasks/TaskList.java

```diff
@@ -4,4 +4,27 @@
 
 public class TaskList {
     private ArrayList<Task> taskArrayList;
+    private static int numberOfTasks = 0;
```

Comment:
Should this really be static?

Reply:
ah, definitely not. I'll go change that.